app/assets/stylesheets/components/comments.scss

```diff
@@ -16,19 +16,34 @@
   &__body {
     margin-top: calc(var(--su-1) * -1);
     padding-left: var(--su-7);
+    padding-bottom: var(--su-2);
+    max-height:330px;
+    overflow: hidden;
 
     a:hover {
       text-decoration: underline; //todo: default for all in-content links
     }
 
-    p {
+    p, div.highlight {
+      margin: 6px auto;
+    }
+
+    * {
       display: -webkit-box;
-      -webkit-line-clamp: 2;
       -webkit-box-orient: vertical;
       overflow: hidden;
+      max-width: 100%;
+      width: auto;
+      object-fit: contain; 
     }
   }
 
+  &__footer {
+    box-shadow: 0px 0px var(--su-2) var(--su-2) var(--story-comments-bg);
```

Comment:
This looks _a little_ funny on some of the themes because it uses a transparent value here so it overlaps weirdly but we can squash that after the fact.